src/plugins/raven/constants.js

```diff
@@ -13,6 +13,11 @@ export const routeIds = {
   DASHBOARD: t('dashboard'),
 };
 
+export const nodeRefs = {
```

Comment:
@nanetteCodes call them collaborationTopics and use GENERAL in the Raven service